AdyenCard/Form/FormCardNumberContainerItem.swift

```diff
@@ -38,7 +38,14 @@ internal final class FormCardNumberContainerItem: FormItem, AdyenObserver {
     internal init(cardTypeLogos: [FormCardLogosItem.CardTypeLogo],
                   style: FormTextItemStyle,
                   localizationParameters: LocalizationParameters?) {
-        self.cardTypeLogos = cardTypeLogos
+        // these 4 US debit brands are not to be displayed
+        // but should be supported so it's done here for now
+        self.cardTypeLogos = cardTypeLogos.filter { logo in
+            logo.type != .accel &&
+                logo.type != .pulse &&
+                logo.type != .star &&
+                logo.type != .nyce
+        }
```

Comment:
```suggestion
       self.cardTypeLogos = cardTypeLogos.minus([.accel, .pulse, .star, .nyce]) 
```

Reply:
Ah very nice! Requires `CardTypeLogo` to conform to `hashable`. Should we add it just for this?